db/schema.rb

```diff
@@ -631,7 +631,7 @@
     t.boolean "sparsity_uplift"
     t.boolean "pupil_premium_uplift"
     t.uuid "delivery_partner_id"
-    t.index ["cpd_lead_provider_id", "participant_profile_id", "declaration_type", "course_identifier", "state"], name: "unique_declaration_index", unique: true, where: "((state)::text = ANY ((ARRAY['submitted'::character varying, 'eligible'::character varying, 'payable'::character varying, 'paid'::character varying, 'clawed_back'::character varying])::text[]))"
+    t.index ["cpd_lead_provider_id", "participant_profile_id", "declaration_type", "course_identifier", "state"], name: "unique_declaration_index", unique: true, where: "((state)::text = ANY (ARRAY[('submitted'::character varying)::text, ('eligible'::character varying)::text, ('payable'::character varying)::text, ('paid'::character varying)::text, ('clawed_back'::character varying)::text]))"
```

Comment:
My version here is from a clean `be rails db:setup`, this shouldn't change. I suspect someone has a local dev database slightly out of sync with the `schema.rb` definition.

Reply:
@peteryates  I've got the same as well after running db:migrate I usually nuke the diff but it's looks like the way the schema is regenerated has changed since. But it does have the same values on the partial index

Reply:
Yeah it's an odd one. I'll dig into it when I've got a min. Will repush this with the diff removed.